content/root/quick-start-guide.textile

```diff
@@ -360,18 +369,28 @@ blang[default].
 blang[python,php].
   The <span lang="python">Python</span><span lang="php">PHP</span> SDK is only available with a REST interface. You will need to use one of the realtime libraries to subscribe to a channel and receive messages. Use the language selector above to select a language with realtime support.
 
-```[javascript](code-editor:quick-start-guide/send-message)
-  const channel = ably.channels.get('quickstart');
-  channel.subscribe('greeting', (message) => {
-    alert(`Received a greeting message in realtime: ${message.data}`);
-  });
+```[javascript]
+// get the channel to subscribe to
+const channel = ably.channels.get('quickstart');
+
+// Subscribe to a channel
+// the promise resolves when the channel is attached (and resolves synchronously
+// if the channel is already attached)
+await channel.subscribe('greeting', (message) => {
+  console.log('Received a greeting message in realtime: ' + message.data)
+});
 ```
 
-```[nodejs](code-editor:quick-start-guide/send-message)
-  const channel = ably.channels.get('quickstart');
-  channel.subscribe('greeting', (message) => {
-    console.log(`Received a greeting message in realtime: ${message.data}`);
-  });
+```[nodejs]
+// get the channel to subscribe to
+const channel = ably.channels.get('quickstart');
+
+// Subscribe to a channel
```

Comment:
Consider using block comments for these multi-line comments? `/* comment ... */`

Reply:
Yes, good idea, reads a little easier in that form.